trace-forward/src/Trace/Forward/Protocol/Limits.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE RankNTypes #-}
+
+module Trace.Forward.Protocol.Limits
+  ( byteLimitsTraceForward
+  , timeLimitsTraceForward
+  ) where
```

Comment:
Both pipe and host:port are supported: https://github.com/input-output-hk/cardano-node/blob/a8d2ad0eb51064bba7e6061149f70c5021c8fa00/trace-forward/src/Trace/Forward/Network/Acceptor.hs#L70-L78

Reply:
Have you considered different protocol limits when using unix sockets / named pipes?

Reply:
@coot I saw such limits here: https://github.com/input-output-hk/ouroboros-network/blob/c82309f403e99d916a76bb4d96d6812fb0a9db81/ouroboros-network/test/Test/Ouroboros/Network/KeepAlive.hs#L93-L94

Reply:
I was asking if you considered to use no limits for unix sockets / named pipes. In that case, consensus is using `runPeer` rather than `runPeerWithLimits`, it also sets `MiniProtocolLimits` (which is a mux ingress queue limit), to [maximumMiniProtocolLimits](https://input-output-hk.github.io/ouroboros-network/ouroboros-network/src/Ouroboros.Network.NodeToClient.html#maximumMiniProtocolLimits).

Reply:
ping

Reply:
Removed.